banyan/resource_service_web.go

```diff
@@ -286,6 +286,13 @@ func WebSchema() (s map[string]*schema.Schema) {
 			Optional:    true,
 			Description: "access tier group which is associated with service",
 		},
+		"tls_sni": {
```

Comment:
@rohitSangamnerkar  add corresponding testcase to cover the scenario

Reply:
added test cases for both scenarios
1 : using default domain TLSSNI
2 : using custom TLSSNI.